src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -378,14 +378,40 @@ bool program_node::has_padded_dependency() const {
     });
 }
 
-std::map<size_t, memory::ptr> program_node::get_const_memory_deps() const {
+bool program_node::is_fused_dep(const primitive_id& dep_id) const {
+    for (auto& fused : get_fused_primitives()) {
+        for (auto& fused_dep : fused.deps) {
+            auto compare_rt = fused_dep.first.compare(dep_id);
+            if (compare_rt == 0) {
+                return true;
+            } else if (compare_rt < 0) {
+                auto cropped_dep_id = dep_id.substr(0, fused_dep.first.size());    // in case of adding '_reorder_'
+                if (fused_dep.first.compare(cropped_dep_id) == 0)
+                    return true;
+            }
+        }
+    }
+
```

Comment:
I think you can simply use fused_primitive_desc::dep_start_idx

Reply:
done by using get_primitive()->input_size()

Reply:
I've changed to use fused_primitive_desc::dep_start_idx instead of get_primitive()->input_size() because get_primitive() doesn't represent original prim.